RELEASE_PROCESS_OPENSEARCH.md

```diff
@@ -0,0 +1,313 @@
+
+#### Docker Build and Scan: 
+
+| Docker | build | scan |
+|----------|----------|----------|
+| OpenSearch  | [Build](https://build.ci.opensearch.org/job/docker-build/3371/)  | [Scan](https://build.ci.opensearch.org/job/docker-scan/1558/artifact/scan_docker_image.txt)   |
+| OpenSearch Dashboards  | [Build](https://build.ci.opensearch.org/job/docker-build/3370/)  | [Scan](https://build.ci.opensearch.org/job/docker-scan/1557/artifact/scan_docker_image.txt)   |
+
+
+#### Docker freeze RC: 
+This to ensure that [check-for-build.jenkinsfile](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/check-for-build.jenkinsfile) wont re-build periodically and override the docker, the RC docker is created with build number.
+
+| Docker Freeze | copy |
+|----------|----------|
+| OpenSearch  | [docker-copy](https://build.ci.opensearch.org/job/docker-copy/466/console)  | 
+| OpenSearch Dashboards  | [docker-copy](https://build.ci.opensearch.org/job/docker-copy/467/console)  | 
+
+#### Performance Test
+| test_workload | sample_run |
+|----------|----------|
+|nyc_taxis |[perf-test](https://build.ci.opensearch.org/job/perf-test/4682/) |
+
+
+#### Windows Integ Test
+
+Currently, the manually tested Windows zip is being evaluated and approved by the plugin teams for sign off. In order to test the Windows distribution, two instances need to be created: one with security features enabled and another without security. Afterward, API calls should be tested by launching the OpenSearch and Dashboard processes through direct execution of the `.bat` file.
+
+#### Broadcast and Communication
+
+Broadcast the release candidate in OpenSearch public slack workspace and the release GitHub issue using below format to gather votes. Following is the sample message that can be posted on the release issue.
+
+  ######  The artifacts can be downloaded from:
+
+  * OpenSearch - #Build-number (Note: Windows version does not have performance analyzer plugin)
+    * arm64 [[manifest](manifest.yml)] [[tar](opensearch-vesion-linux-arm64.tar.gz)] [[rpm](opensearch-vesion-linux-arm64.rpm)][[deb](opensearch-version-linux-arm64.deb)]
+    * x64 [[manifest](manifest.yml)] [[tar](opensearch-version-linux-x64.tar.gz)] [[rpm](opensearch-version-linux-x64.rpm)] [[deb](opensearch-version-linux-x64.deb)] [[windows](opensearch-version-windows-x64.zip)]
+  * OpenSearch Dashboards - #Build-number
+    * arm64 [[manifest](manifest.yml)] [[tar](opensearch-dashboards-vesion-linux-arm64.tar.gz)] [[rpm](opensearch-dashboards-vesion-linux-arm64.rpm)][[deb](opensearch-dashboards-version-linux-arm64.deb)]
+    * x64 [[manifest](manifest.yml)] [[tar](opensearch-dashboards-version-linux-x64.tar.gz)] [[rpm](opensearch-dashboards-version-linux-x64.rpm)] [[deb](opensearch-dashboards-version-linux-x64.deb)] [[windows](opensearch-dashboards-version-windows-x64.zip)]
```

Comment:
Probably add example url of comments from previous release.

Reply:
Thanks added to ref https://github.com/opensearch-project/opensearch-build/issues/3434#issuecomment-1571201919.